scene/main/viewport.cpp

```diff
@@ -491,6 +491,7 @@ void Viewport::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_WM_WINDOW_FOCUS_OUT: {
+			_gui_cancel_tooltip();		
```

Comment:
You now have trailing whitespace on this line (two tabs at the end of the line):

![image](https://user-images.githubusercontent.com/4701338/203938572-7077d5a7-64bf-497a-ad84-6a80267f059b.png)

You can set up clang-format locally in your IDE, or via our pre-commit hooks https://github.com/godotengine/godot/tree/master/misc/hooks, to make sure that files comply with style requirements.

Or configure your IDE to strip trailing whitespace, which would cover at least this type of error.

Reply:
Sorry for that. I've installed the hooks you suggested and the most recent update has made it through clang format on my end. 

on a slight side note the clang format script doesn't seem to complain about the NL you originally mentioned but it does automatically strip the WS.

Hope everything is good this time.

Reply:
The newline wasn't a problem, the problem is that there was whitespace on the newline :)